cl/search/feeds.py

```diff
@@ -97,6 +97,38 @@ def item_title(self, item):
     def item_categories(self, item):
         return [item['status'], ]
 
+    def item_enclosure_url(self, item):
+        try:
+            path = item['local_path']
+            if not path.startswith('/'):
+                return '/%s' % (path,)
+            return path
+        except:
+            return None
+
+    def item_enclosure_length(self, item):
+        try:
+            file_loc = os.path.join(
+                settings.MEDIA_ROOT,
+                item['local_path'].encode('utf-8')
+            )
+            return os.path.getsize(file_loc)
+        except:
+            return None
+
+    def item_enclosure_mime_type(self, item):
+        try:
+            file_loc = os.path.join(
+                settings.MEDIA_ROOT,
+                item['local_path'].encode('utf-8')
+            )
+            mime = magic.from_file(file_loc, mime=True)
+            if mime.startswith('inode'):
+                return None
+            return mime
```

Comment:
Easily doable and I think the only consequence being if it's for some accurate, it should only cause a minor hiccup on how the client tries to handle the response, but shouldn't block the download. I'll assume that:

* There's a file extension and it's accurate.
* If not I'll check the parent directory (typically 'pdf/') and assume that's accurate.
* All else fails, default to `application/octet-stream` so the client should properly see it as a downloadable binary file and not try to render the byte stream. (Only side effect could be a text-based file slipping through mime-type guessing and ending up downloaded and not displayed natively in the browser.)

Thinking it best to use the mimetypes based on file extensions as shown on this [StackOverflow](http://stackoverflow.com/questions/4212861/what-is-a-correct-mime-type-for-docx-pptx-etc) post.